meeting-minutes/2024-10-16.md

```diff
@@ -0,0 +1,147 @@
+ 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 9 or more of the 17 voting members (including the officers).
+
+| First Name | Last Name  | Company                                                     | Role(s)                   | Present |
+|:-----------|:-----------|:------------------------------------------------------------|:--------------------------|:--------|
+| Adrian     | Diglio     | Microsoft                                                   | Voting Member             | NO      |
+| David      | Kemp       | National Security Agency                                    | Member                    | NO      |
+| Denny      | Page       | Individual                                                  | Voting Member             | NO      |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member             | NO      |
+| Feng       | Cao        | Oracle                                                      | Member                    | YES     |
```

Comment:
@sthagen What does "Voting Member" mean? Is that the indication that the person has voting rights?

Reply:
@tschmidtb51 the table entries attempt to match the roster entries. Voting members are members bearing voting rights as distinction from members (only). Co-chairs and secretary are also members with voting rights. So, in our case the count of members labeled "voting member" plus 3 is the total count of members with voting rights.
Does this help?

Reply:
Good to resolve this @tschmidtb51?

Reply:
@sthagen Thank you.
@justmurphy: Yes. It is resolved.